junit-jupiter-engine/junit-jupiter-engine.gradle.kts

```diff
@@ -39,6 +41,9 @@ tasks {
 					org.junit.platform.engine;\
 						org.junit.platform.engine='junit-jupiter';\
 						version:Version="${'$'}{version_cleanup;${project.version}}"
+				Require-Capability:\
+					org.junit.platform.launcher;\
+						filter:='(&(org.junit.platform.launcher=junit-platform-launcher)(version>=${'$'}{version_cleanup;${rootProject.property("platformVersion")!!}})(!(version>=${'$'}{versionmask;+;${'$'}{version_cleanup;${rootProject.property("platformVersion")!!}}})))'
```

Comment:
I think I'm ok with the way you've done the Req/Prov on the `org.junit.platform.launcher` namespace (although it's basically the same as a `RequireBundle`). I can see that it might make bndtools/bnd#3795 practically redundant (though I think the theoretical argument for something like bndtools/bnd#3795 is compelling - generally speaking, if you require a particular bundle to be present at runtime, then you must also require its requirements).

Reply:
Yes, I can see the value in decoupling the capability namespace from the bsn.